dbld/rpm

```diff
@@ -8,6 +8,10 @@ USER=`whoami`
 RPMBUILD=/home/${USER}/rpmbuild
 RPMBUILD_SOURCES=$RPMBUILD/SOURCES
 
+# CzP's spec file expect it in the pwd
+# where the build was initiated
+cp packaging/rhel/syslog-ng.* /build
+
```

Comment:
My understanding is that they need to be in the rpmbuild source directory. Isn't that the case?

Reply:
It is looking for the files where rpmbuild is initiated. In our case it is '''/build'''

Reply:
I was experimenting with it as well.
rpmbuild is using the dist tarball, isn't it?
Why don't we include it our Makefile instead of this copy? I have tried it out and it works, it would be similar to what our debian packaging does.